includes/blocks/class-sensei-course-results-block.php

```diff
@@ -128,20 +144,30 @@ private function render_total_grade( $course_id ) {
 	/**
 	 * Render the course title.
 	 *
-	 * @param int $course_id Course ID.
+	 * @param int   $course_id  Course ID.
+	 * @param array $structure  Course structure information.
+	 * @param array $attributes The block attributes.
 	 *
 	 * @return string HTML for the course title.
 	 */
-	private function render_course_title( $course_id ) {
+	private function render_course_title( $course_id, $structure, $attributes ) {
 		$content      = [];
 		$course_title = $course_id ? get_the_title( $course_id ) : '';
 
 		if ( $course_title ) {
-			$content[] = '<h2 class="wp-block-sensei-lms-course-results__title">';
+			$content[] = '<h2 class="wp-block-sensei-lms-course-results__course-title">';
 			$content[] = $course_title;
 			$content[] = '</h2>';
 		}
 
+		// Render separator for courses that don't have any modules.
+		$type_counts = array_count_values( array_column( $structure, 'type' ) );
```

Comment:
Since we're making this same check for modules and lessons, would it be worth adding a separate method for this check?

Reply:
Done in https://github.com/Automattic/sensei/pull/4279/commits/48baea027b0e9901ac9eef757043040afa53d8cc.